app/views/docs/realtime.phtml

```diff
@@ -0,0 +1,341 @@
+        <h3>Web</h3>
+        <div class="ide margin-bottom" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>let sdk = new Appwrite();
+
+const unsubscribe = sdk.subscribe('files', response => {
+    // Callback will be executed on changes for all files.
+    console.log(response);
+});
+
+// Closes the subscription.
+unsubscribe();</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Flutter</h3>
+
+        <div class="ide margin-bottom" data-lang="dart" data-lang-label="Flutter SDK">
+            <pre class="line-numbers"><code class="prism language-dart" data-prism>final client = Client();
+final realtime = Realtime(client);
+final subscription = realtime.subscribe('files');
+
+subscription.stream.listen((response) {
+    // Callback will be executed on changes for all files.
+    print(response);
+})
+
+// Closes the subscription.
+subscription.close();</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Kotlin</h3>
+        <div class="ide margin-bottom" data-lang="kotlin" data-lang-label="Kotlin SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>val client = Client()
+val realtime = Realtime(client)
+
+val subscription = realtime.subscribe("files", callback = { param ->
+    // Callback will be executed on changes for all files.
+    print(param.toString())
+})
+
+// Closes the subscription.
+subscription()</code></pre>
+        </div>
+    </li>
+</ul>
+
+<h2><a href="/docs/realtime#payload" id="payload">Payload</a></h2>
+
+<p>The payload from the subscription will contain following properties:</p>
+
+<table class="vertical full args">
+    <thead>
+        <tr>
+            <td style="width: 140px">Name</td>
+            <td style="width: 120px">Type</td>
+            <td>Description</td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td data-title="Name: ">
+                event
+            </td>
+            <td data-title="Type: ">
+                string
+            </td>
+            <td data-title="Description: ">
+                The <a href="/docs/webhooks#events">system event</a> that triggered this update.
+            </td>
+        </tr>
+        <tr>
+            <td data-title="Name: ">
+                channels
+            </td>
+            <td data-title="Type: ">
+                string[]
+            </td>
+            <td data-title="Description: ">
+                An array of <a href="/docs/realtime#channels">channels</a> that can receive this message.
+            </td>
+        </tr>
+        <tr>
+            <td data-title="Name: ">
+                timestamp
+            </td>
+            <td data-title="Type: ">
+                integer
+            </td>
+            <td data-title="Description: ">
+                The UNIX timestamp from the server to ensure consistency across all client platforms and real-time technologies.
```

Comment:
For beginners, it might be useful to include a link to https://en.wikipedia.org/wiki/Unix_time on the term `UNIX timestamp`. Use `rel="noopener"` and `target="_blank"` for the link tag.